egg-herbie/src/math.rs

```diff
@@ -103,6 +103,17 @@ pub struct ConstantFold {
     pub prune: bool,
 }
 
+impl Clone for ConstantFold {
+    fn clone(&self) -> Self {
+        let unsound = AtomicBool::new(self.unsound.load(Ordering::SeqCst));
+        Self {
+            unsound,
+            constant_fold: self.constant_fold,
+            prune: self.prune,
+        }
+    }
+}
```

Comment:
Why? Does it matter? Is this just because we need it somehow for serialize?

Reply:
This matters for the Herbie-side scheduler: we actually clone the runner. The analysis didn't implement the `Clone` trait.